packages/ember-glimmer/lib/component-managers/capabilities.ts

```diff
@@ -0,0 +1,30 @@
+import { assert } from 'ember-debug';
+import { assign } from 'ember-utils';
+
+// because we're not yet compatible with latest Glimmer
+// we lack typings in Ember.js.
+// steal capabilities interface from http://bit.ly/2gMdhv5 for now
+export interface ComponentCapabilities {
+  dynamicLayout: boolean;
+  dynamicTag: boolean;
+  prepareArgs: boolean;
+  createArgs: boolean;
+  attributeHook: boolean;
+  elementHook: boolean;
+}
+
+// default capabilities values stolen from http://bit.ly/2zfZQyz
+export const DEFAULT_CAPABILITIES: ComponentCapabilities = {
+  dynamicLayout: true,
+  dynamicTag: true,
+  prepareArgs: true,
+  createArgs: true,
+  attributeHook: false,
+  elementHook: false
+};
+
+export default function capabilities(specifier: string, features: Object): Object {
```

Comment:
what should `specifier` be here? just ember versions like `2.*` and so on?

Reply:
Yes, lets use Ember versions for this.